Model/LogicModelApi.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+using BallSimulator.Logic;
+using BallSimulator.Presentation.Model;
+
+namespace BallSimulator.Presentation.Model
+{
+    public abstract class LogicModelApi
+    {
+        protected Observer _observer;
+
+
+        public delegate void Observer(IEnumerable<BallModel> ballModels);
+        public abstract void NotifyUpdate();
+        public abstract void SetObserver(Observer modelObserver);
+
+        public abstract void SpawnBalls(int count);
+        public abstract void Start();
+        public abstract void Stop();
+        public abstract IEnumerable<BallModel> MapBallToBallModel();
+
+        public static LogicModel CreateLogicModelApi(LogicAbstractApi logic = default)
```

Comment:
Why do you need a factory method in an abstract class?
As it is now, it implies you will never have another implementation of this abstract class, and therefore it probably should not exist.

Sorry for barging in on your PR, I was bored so I looked up some open PRs on public repos.